v3/plugins/che-incubator/typescript/1.35.1/meta.yaml

```diff
@@ -10,6 +10,9 @@ icon: https://raw.githubusercontent.com/microsoft/vscode/1.35.1/extensions/types
 repository: https://github.com/Microsoft/vscode
 category: Programming Languages
 firstPublicationDate: '2019-06-20'
+deprecate:
```

Comment:
this file should be removed as it will be generated from the che-theia-plugins.yaml file